### ISSUE TITLE
wishlist: hstore/array support for TextSymbolizer's list

### ISSUE THREAD

**Issue by tobwen** [mapnik/mapnik-support#56]
I'm not able to provide code for this, but it would be pretty useful to give TextSymbolizer's list function the ability to access hstore or arrays. Right now, you need to create a column for each possible abbreviation. Using hstore or array you'd be much more flexible in choosing an alternative.

**Comment by talaj** [comment]
Some movement in this direction is https://github.com/mapnik/mapnik/issues/1113.

**Comment by tobwen** [comment]
Exactly! 2012 was the year, I talked to herm about this idea on IRC :)
